fiftyone/server/routes/samples.py

```
@@ -0,0 +1,185 @@
+            after = int(after) if after is not None else None
+            count = int(data.get("count") or 0)
+        except (TypeError, ValueError):
+            return JSONResponse(
+                {"error": "'after' and 'count' must be integers"},
+                status_code=400,
+            )
+
+        dataset = await run_sync_task(get_dataset, dataset_id)
+        sample_filter = _parse_group_slice_filter(data.get("filter"))
+
+        def _build():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_build`

(ANN202)